### PR TITLE
Internationalisation: Add basic e2e test to ensure languages load correctly

### PR DESCRIPTION
**Pull request by ashharrison90**
**What is this feature?**

- adjusts the pseudo locale id to be just `pseudo`
  - latest version of i18next has some code that attempts to verify the language code is correct if it contains a `-`
  - `pseudo-LOCALE` fails this check and doesn't work anymore
- adds an e2e test to verify prod translations load correctly
- disables inputs whilst loading preferences: 
image

**Why do we need this feature?**

- bumping i18next broke pseudo translations (see [this PR](https://github.com/grafana/grafana/pull/96392))
- prevent regressions in i18n
- fix pseudo translations

**Who is this feature for?**

- grafana devs

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

File: e2e/various-suite/verify-i18n.spec.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  };
+
+  it('loads all the languages correctly', () => {
+    cy.intercept('GET', '/api/user/preferences').as('preferences');
+    cy.visit('/profile');
+
+    cy.wrap(Object.entries(languageMap)).each(([language, label]: [string, string]) => {
+      // TODO investigate why we need to wait for 6 (SIX!) calls to the preferences API
+      cy.wait('@preferences');
+      cy.wait('@preferences');
```
Comment (joshhunt) [inline e2e/various-suite/verify-i18n.spec.ts:25]: yikes

File: e2e/various-suite/verify-i18n.spec.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      cy.wait('@preferences');
```
Comment (joshhunt) [inline e2e/various-suite/verify-i18n.spec.ts:26]: yikes

**Review by joshhunt — APPROVED** [review]

File: public/app/core/internationalization/constants.ts [hunk 3]
```diff
@@ -7,7 +7,7 @@ export const SPANISH_SPAIN = 'es-ES';
 export const GERMAN_GERMANY = 'de-DE';
 export const BRAZILIAN_PORTUGUESE = 'pt-BR';
 export const CHINESE_SIMPLIFIED = 'zh-Hans';
-export const PSEUDO_LOCALE = 'pseudo-LOCALE';
+export const PSEUDO_LOCALE = 'pseudo';
```
[the remark this answers is not in the document]
Comment (ashharrison90) [reply public/app/core/internationalization/constants.ts:10]: yeah i figured that would happen as well. it just falls back to default from my testing, which i think is fine

**Review by joshhunt — APPROVED** [review]

**Review by joshhunt — APPROVED** [review]
polished!

**Pull request merged**